hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceManager.java

```diff
@@ -244,7 +286,7 @@ private void adoptAbandonedQueues() {
    * </ol>
    * @param peerId the id of replication peer
    */
-  public void addPeer(String peerId) throws IOException {
+  void addPeer(String peerId) throws IOException {
```

Comment:
Funny, I want it to stay public in my current work (meta region replicas). Can move it back in my patch.

Reply:
I want to call it on Region open. Let me see if I can move where I make the call.